ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/preexecution/PreExecutingSubmissionValidator.scala

```diff
@@ -30,22 +19,22 @@ import scala.concurrent.{ExecutionContext, Future}
 /**
   * Validator for pre-executing submissions.
   */
-class PreExecutingSubmissionValidator[ReadSet, WriteSet](
+class PreExecutingSubmissionValidator[StateValue, ReadSet, WriteSet](
```

Comment:
I'd prefer if this type parameter implied that it is wrapping a DAML state value. E.g., a more specific name may be `DecoratedDamlStateValue`. Alternatively, please add some code docs.

Reply:
I've added some docs to explain. Thank you for the nudge to do so. :grinning: